src/components/CourseForm/index.test.tsx

```diff
@@ -60,6 +79,281 @@ describe('Course Form Course Create Tests', () => {
     createdById: '30',
     image: 'http://test-image.com',
   };
+  const courseStart = new Date(Date.now() + 24 * 60 * 60 * 1000);
+  const courseEnd = new Date(Date.now() + 2 * 24 * 60 * 60 * 1000);
+  const oneDayBeforeStart = new Date();
+
+  const course = {
+    id: '1234',
+    createdById: '30',
+    name: 'New course',
+    description: 'A test course',
+    summary: 'All you ever wanted to know about testing!',
+    startDate: courseStart,
+    endDate: courseEnd,
+    lastEnrollDate: oneDayBeforeStart,
+    lastCancelDate: oneDayBeforeStart,
+    maxStudents: 55,
+    tags: [
+      { id: '1', name: 'Testing' },
+      { id: '2', name: 'Git' },
+    ],
+    image: 'http://test-image.com',
+  };
+
+  const setNativeValue = (element: HTMLInputElement, value: string) => {
+    // sets date and time input values
+    let lastValue = element.value;
+    element.value = value;
+
+    let event = new Event('input', { bubbles: true });
+
+    (event as any).simulated = true;
+
+    let tracker = (element as any)['_valueTracker'];
+    if (tracker) {
+      tracker.setValue(lastValue);
+    }
+    element.dispatchEvent(event);
+  };
+
+  it('Form wont be submitted if the required fields are empty', async () => {
```

Comment:
Here it would be better to write it('wont submit a form if the required fields are empty') or test('Form wont be submitted if the required fields are empty'). In Jest 'it(..)' and 'test(..)' are functionally the same and the purpose is to write a readable English sentence that describes the test.